src/index.ts

```diff
@@ -10,40 +21,21 @@ export = {
         'no-undef': 'off',
 
         // Rules replaced by @typescript-eslint versions:
-        camelcase: 'off',
-        indent: 'off',
-        'no-array-constructor': 'off',
-        'no-unused-vars': 'off',
+        ...Object.fromEntries(equivalents.map((name) => [name, 'off'])),
         'no-use-before-define': 'off',
-        'no-useless-constructor': 'off',
-        quotes: 'off',
 
         // @typescript-eslint versions of Standard.js rules:
-        '@typescript-eslint/camelcase': ['error', { properties: 'never' }],
-        '@typescript-eslint/indent': ['error', 2, {
-          SwitchCase: 1,
-          VariableDeclarator: 1,
-          outerIIFEBody: 1,
-          MemberExpression: 1,
-          FunctionDeclaration: { parameters: 1, body: 1 },
-          FunctionExpression: { parameters: 1, body: 1 },
-          CallExpression: { arguments: 1 },
-          ArrayExpression: 1,
-          ObjectExpression: 1,
-          ImportDeclaration: 1,
-          flatTernaryExpressions: false,
-          ignoreComments: false,
-          ignoredNodes: ['TemplateLiteral *']
+        ...Object.fromEntries(
+          equivalents.map((name) => [`@typescript-eslint/${name}`, standardRules[name]])
+        ),
+        '@typescript-eslint/no-use-before-define': ['error', {
+          functions: false,
+          classes: false,
+          variables: false,
+          typedefs: false // Only the TypeScript rule has this option.
         }],
-        '@typescript-eslint/no-array-constructor': 'error',
-        '@typescript-eslint/no-empty-function': 'error',
-        '@typescript-eslint/no-unused-vars': ['error', { vars: 'all', args: 'none', ignoreRestSiblings: true }],
-        '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: false, typedefs: false }],
-        '@typescript-eslint/no-useless-constructor': 'error',
-        '@typescript-eslint/quotes': ['error', 'single', { avoidEscape: true, allowTemplateLiterals: false }],
-        '@typescript-eslint/require-await': 'error',
```

Comment:
Most of this block is programmatically obtained from the real values from Standard.